Objects/memoryobject.c

```diff
@@ -1338,16 +1339,17 @@ zero_in_shape(PyMemoryViewObject *mv)
 static PyObject *
 memory_cast(PyMemoryViewObject *self, PyObject *args, PyObject *kwds)
 {
-    static char *kwlist[] = {"format", "shape", NULL};
+    static char *kwlist[] = {"format", "shape", "readonly", NULL};
     PyMemoryViewObject *mv = NULL;
     PyObject *shape = NULL;
+    PyObject *readonly = NULL;
     PyObject *format;
     Py_ssize_t ndim = 1;
 
     CHECK_RELEASED(self);
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O|O", kwlist,
-                                     &format, &shape)) {
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O|OO", kwlist,
```

Comment:
I wonder if it makes sense for the `readonly` argument to be keyword-only. @skrah may have an opinion on this.

Reply:
Yes, intuitively I think it should be keyword-only.